index.html

```diff
@@ -57,6 +57,308 @@ <h2>I'm a Software Developer</h2>
+            <ul class="lang-list">
+              <li><a href="#">JavaScript</a></li>
+              <li><a href="#">Ruby</a></li>
+              <li><a href="#">HTML</a></li>
+              <li><a href="#">CSS</a></li>
+            </ul>
+          </div>
+        </div>
+        <div class="lang-container">
+          <div class="lang-img">
+            <img src="./pics/rec2.svg" alt="rec" />
+            <h4>Frameworks</h4>
+          </div>
+          <div>
+            <ul class="lang-list">
+              <li><a href="#">React.js</a></li>
+              <li><a href="#">Ruby on Rails</a></li>
+              <li><a href="#">RSpec</a></li>
+              <li><a href="#">Capybara</a></li>
+            </ul>
+          </div>
+        </div>
+        <div class="lang-container">
+          <div class="lang-img">
+            <img src="./pics/rec3.svg" alt="rec" />
+            <h4>skills</h4>
+          </div>
+          <div>
+            <ul class="lang-list">
+              <li><a href="#">Database Management</a></li>
+              <li><a href="#">Version Control</a></li>
+              <li><a href="#">CLI</a></li>
+              <li><a href="#">API Design</a></li>
+              <li><a href="#">Web Design</a></li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section>
+      <div class="interest">
+        <div class="intrest-description">
+          <h5>
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h5>
+        </div>
+        <div class="contact">
+          <div class="contact-input">
+            <input type="text" placeholder="Full Name" name="name" required />
+          </div>
+          <div class="contact-input">
+            <input
+              type="email"
+              placeholder="Email Address"
+              name="email"
+              required
+            />
+          </div>
+          <div class="info">
+            <textarea placeholder="Write something ..." rows="6"></textarea>
+          </div>
+        </div>
+        <div class="contact-btn">
+          <div class="story-btn">
+            <span><a href="#">Get in touch</a></span>
+          </div>
+        </div>
+      </div>
+    </section>
+    <footer>
+      <div class="footer-content">
+        <ul class="social-media footer-social">
+          <li>
+            <a href="#"><img src="./pics/github.svg" alt="github" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./pics/link.svg" alt="linkedin" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./pics/yah.svg" alt="yep" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./pics/twitter.svg" alt="twitter" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./pics/medium.svg" alt="medium" /></a>
+          </li>
+        </ul>
+        <hr />
+      </div>
+    </footer>
```

Comment:
- [ ] Kindly remove this section as it's not part of the requirement for this milestone. (You'll start working on the form in more detail next week 🏁) you can keep it on some other local file and add it to your next PR next week. ✔
- Please restrain from adding contents that are part of future milestones as this will cause much confusion when checking the "files changed" section later on for the particular milestone.

Reply:
done :heavy_check_mark: